Makefile

```diff
@@ -125,6 +125,12 @@ delete-tag:
 	 	git tag -d "$${dir:2}/$${TAG}" ); \
 	done
 
+.PHONY: prepare-tag
+prepare-tag:
+	@[ "${TAG}" ] || ( echo ">> env var TAG is not set"; exit 1 )
+	gsed -i 's#\(gomod: "github.com/SumoLogic/sumologic-otel-collector/.*\) v0.0.0-00010101000000-000000000000#\1 ${TAG}#g' \
+		otelcolbuilder/.otelcol-builder.yaml
+
```

Comment:
where is it being used?

Reply:
CI release job. Also documented in `Dev.md`

Reply:
:+1: my eyes don't see grey color :see_no_evil: